stacks/api/bucket/bucket-data.controller.ts

```diff
@@ -16,9 +16,9 @@ import {Schema} from "@spica-server/core/schema";
 import {FilterQuery, MongoError, ObjectId, OBJECT_ID} from "@spica-server/database";
 import {ActionGuard, AuthGuard} from "@spica-server/passport";
 import * as locale from "locale";
-import {BucketDocument} from "./bucket";
+import {BucketDocument} from "@spica-server/bucket/services/bucket";
```

Comment:
We should reexport the public references from their individual index.ts file. Thus we can import them like `@spica-server/bucket/services` instead of `@spica-server/bucket/services/bucket`.